unit_tests/test_stg_ga4__page_conversions.py

```diff
@@ -23,6 +23,12 @@
 
 
 class TestPageConversions:
+    # Update project name to ga4 so we can call macros with ga4.macro_name
+    @pytest.fixture(scope="class")
+    def project_config_update(self):
+        return {
+            "name": "ga4"
+        }
```

Comment:
I thought adding this element like in the other tests on models using macros was going to fix the issue but it doesn't.